src/components/demo-iframe/demo-iframe.js

```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 import ChevronousText from '@mapbox/mr-ui/chevronous-text';
 import MapWrapper from '../map-wrapper/map-wrapper';
 
-export default class DemoIframe extends React.Component {
+export default class DemoIframe extends React.PureComponent {
```

Comment:
I just checked this example on `main` and it shows the same thing. I can't remember what this demo wants to achieve since it sends an intentionally incorrect access token. I'll ticket this and follow up!